src/templates/users/user_history.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>User Historical Information</h2>
+  <ul>
+    <li>Username: {{ user.name }}</li>
```

Comment:
Because there is a username field in the model (even though it's currently set to none), I would change this to Name instead of Username.